docs/package.json

```diff
@@ -61,7 +61,7 @@
     "lz-string": "^1.4.4",
     "markdown-to-jsx": "^7.1.7",
     "marked": "^4.0.17",
-    "next": "12.2.0",
+    "next": "^12.2.5",
```

Comment:
We have two different Next.js app, I'm keeping this one in sync with the docs-infra

Reply:
Ideally we'd decouple `@mui/monorepo` as well